lib/customer.rb

```diff
@@ -0,0 +1,45 @@
+require "csv"
+require "ap"
+require "pry"
+
+class Customer
+  attr_reader :id, :email, :address
+  @@customers = []
+  @@found_customer = nil
```

Comment:
You don't end up using `@@found_customer` again... and to be honest, that's probably a good thing! If you have a concept of a "found customer," it's probably just good for a specific situation, and doesn't need to be held as a class variable.